pyproject.toml

```diff
@@ -18,7 +18,7 @@ dependencies = [
     "pygithub >= 1.55",
     # pyclip with this patch https://github.com/spyoungtech/pyclip/pull/21
     "pyclip @ https://github.com/mgaitan/pyclip/archive/refs/heads/best_binary.zip",
-    "python-magic >= 0.4.3",
+    "python-magic >= 0.4.27",
```

Comment:
why this?

Reply:
And I think that because of that doesn't install the library in brew

Reply:
for our purpose, there is any real difference with this change, because if python-magic is not installed (for instance in a fresh virtualenv like the created by pipx), then it will install the latest version anyway, but if the tool is installed among other components in a virtualenv and magic is already installed, we won't force it to be the latest one (and 4.3 is enough for us). 

Also, 0.4.27 doesnt fix automatically the dependency in mac. 

We should workaround that problem by encapsulating the importt in a function, and making it optionally

```python
def get_extension(content):
     try:
         import magic
     except ImportError as e:  
         logger.warning(str(e))   # or print  + rich in yellow 
         return ""            
         # Optionally could we detect a few formats in our own?
         # for instance, images are easy https://stackoverflow.com/a/27670182/811740
     else: 
          return guess_extension(magic.from_buffer(content, mime=True))              
```
      
    
Also, we should mention the way to install the lib in our README
https://github.com/ahupp/python-magic#osx